Resources/doc/reference/installation.rst

```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle
+    $ composer require sonata-project/media-bundle
```

Comment:
So you'd like dev-kit to have a refactoring command, right ? With two arguments, like "shell-command" and "file-pattern". Could be handy.